extensions/amp-story/1.0/amp-story.js

```diff
@@ -460,9 +460,9 @@ export class AmpStory extends AMP.BaseElement {
         );
       });
     }
-    if (isExperimentOn(this.win, 'amp-story-load-first-page-only')) {
+    if (isExperimentOn(this.win, 'story-load-first-page-only')) {
       Services.performanceFor(this.win).addEnabledExperiment(
-        'amp-story-load-first-page-only'
+        'story-load-first-page-only'
```

Comment:
Nit: This appears a bit strange, why does `amp-story` read for the value of an experiment followed by only setting the same value for elsewhere?

Reply:
Please let us know if we're missing something:
The CSI pipeline doesn't track experiments by defaults, we need to specifically enable them through this method so they appear in our dashboards. We'll want to monitor the LCP improvements for this experiment before launching fully.

Reply:
Don't think you're missing anything, but we should revisit this later on.

CSI should automatically insert all tests so we don't have to remember to do this.